dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -1000,3 +1000,20 @@ CWWKD1100.cursor.requires.sort.explanation=Cursor-based pagination requires \
 CWWKD1100.cursor.requires.sort.useraction=Remove the ORDER BY clause from the \
  Query annotation value if one is found there. Add an OrderBy annotation or a \
  repository method parameter to specify the sort criteria.
+
+CWWKD1101.attr.subset.mismatch=CWWKD1101E: The {0} return type of the \
+ {1} method of the {2} repository attempts to return results as the subset \
+ of entity attributes that are represented by the {3} Java record type, \
+ expecting the {4} entity to have attributes named: {5}. To return a subset \
```

Comment:
To return a subset of entity attributes as a Java record, ensure that the record component names are within the following set of valid attribute names for the {4} entity: {6}.

Reply:
Updating as suggested